src/App.tsx

```diff
@@ -1,8 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
-);
+import * as api from './api';
+import PostsList from './PostList';
+
+const App: React.FC = () => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
Please add typing

Reply:
additionally, I would think about the naming, so that we do not get mixed up with isLoading I would try to call this element maybe isReady, consider this please.